demisto_sdk/commands/download/tests/downloader_test.py

```diff
@@ -61,8 +61,8 @@ def setup(self):
         tests_path = self.tmp_path / 'tests'
         tests_env_path = tests_path / 'tests_env'
         tests_data_path = tests_path / 'tests_data'
-        shutil.copytree(src='demisto_sdk/commands/download/tests/tests_env', dst=str(tests_env_path))
-        shutil.copytree(src='demisto_sdk/commands/download/tests/tests_data', dst=str(tests_data_path))
+        shutil.copytree(src='/Users/ohaim/dev/demisto/demisto-sdk/demisto_sdk/commands/download/tests/tests_data', dst=str(tests_env_path))
+        shutil.copytree(src='/Users/ohaim/dev/demisto/demisto-sdk/demisto_sdk/commands/download/tests/tests_data', dst=str(tests_data_path))
```

Comment:
You can't use your local paths, the paths should be within the demisto-sdk repo.
```suggestion
        shutil.copytree(src='demisto_sdk/commands/download/tests/tests_data', dst=str(tests_data_path))
```